solution/local.defaults.properties

```diff
@@ -0,0 +1 @@
+GOOGLE_MAPS_API_KEY=DEFAULT_API_KEY
```

Comment:
MAPS_API_KEY (without the GOOGLE_ prefix)

Reply:
Also add PLACES_API_KEY=DEFAULT_API_KEY like the [documentation for Places SDK](https://developers.google.com/maps/documentation/places/android-sdk/config#get-an-api-key)